modules/authentication/portal-authentication-token/README.adoc

```diff
@@ -0,0 +1,28 @@
+= portal-authentication-token
+
+== What is it?
+Provides some convenience structures for dealing with JTW-Token.
+
+It is essentially a wrapper around the types from io.smallrye:smallrye-jwt.
+
+The core functionality is the simplified configuration of checking the signature of a given token, by looking up the corresponding public-keys from an oauth-server, tested with keycloak.
+
+== Maven Coordinates
+
+[source, xml]
+----
+    <dependency>
+        <groupId>de.cuioss.portal.authentication</groupId>
+        <artifactId>portal-authentication-token</artifactId>
+    </dependency>
+----
```

Comment:
is it a subset of `oauth`?

Reply:
Aargh. I wondered about that myself. Actually it is only used in the context of oauth. But jwt-token, like smallry-jwt is neutral to oauth, therefore i made a separate module. Especially I need it on an other project: [nifi-extesnisons](https://github.com/cuioss/nifi-extensions): Ther I need it solely for the offline validation